src/nano_contracts/serializer.ts

```diff
@@ -234,7 +245,7 @@ class Serializer {
    * Serialize a tuple of values
    *
    * @param value List of values to serialize
-   * @param typeStr Comma separated list of types e.g. `str,int,VarInt`
+   * @param typeArr Comma separated list of types e.g. `str,int,VarInt`
```

Comment:
chore: Same here. The typescript type specifies an array, but the description says it's a comma separated string.

Reply:
Fixed